content/docs/rtl-support/index.md

```diff
@@ -0,0 +1,114 @@
+# RTL support
+The goal is to have full RTL support for all of ionic.
+If you don't need RTL support, you can set `$include-rtl: false`, and your css bundle will be smaller.
```

Comment:
@piotrowski disabling/enabling RTL is probably a micro-optimization that will not have any measurable effect in your app